client/src/app/dashboard/crawler/crawler.component.html

```diff
@@ -1,6 +1,12 @@
-<div class="container-fluid">
-    <h2> Crawler Details</h2>
+<h2> Crawler Details</h2>
+<div class="container">
     <div class="row">
-        <app-table-all [data_array] = "table_data"> </app-table-all>
+    	
+        <app-table-all [data_array] = "data_array"> </app-table-all>  
+        
    </div>
-</div>
\ No newline at end of file
+   <div class="button-row">
+		<button mat-raised-button color="primary" (click)="openDialog()">Delete Selected</button>
```

Comment:
if you have a look at your crawler, webapi, and webapp component you will notice that you basically have written the same code there regarding the "button-row" both in the html and the typescript.
I think it would be a good idea to extract this as a single component (similar to what you did with the table-all component), which then provides the result of the input dialogs back to the specific component class. 
This would increase the reusability and maintainability of the code further.